pages/Pokémon Dollars/overview.html

```diff
@@ -0,0 +1,81 @@
+                <td>(AC/LI) Average</td>
+                <td>Average Pokedollars per defeated Pokémon with either Amulet Coin or Lucky Incense active</td>
+            </tr>
+            <tr>
+                <td>(AC*LI) Average</td>
+                <td>Average Pokedollars per defeated Pokémon with both Amulet Coin and Lucky Incense active</td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+<br/>
+<div>
+    <h3>Regional Rate Tables</h3>
+    <div data-bind="foreach: GameHelper.enumNumbers(GameConstants.Region).filter((r) => r != GameConstants.Region.final && r != GameConstants.Region.none)">
+        <h3><a data-bind="text: GameConstants.camelCaseToString(GameConstants.Region[$data]), attr: { href: `#!Regions/${GameConstants.camelCaseToString(GameConstants.Region[$data])}` }">Region</a></h3>
+    
+        <div class="table-responsive">
+            <table class="table table-hover table-striped table-bordered">
+                <thead class="thead-dark">
+                    <tr>
+                        <th>Route/Gym</th>
+                        <th>Max Health</th>
+                        <th>Average</th>
+                        <th>(AC/LI) Average</th>
+                        <th>(AC*LI) Average</th>
+                    </tr>
+                </thead>
+                <tbody data-bind="foreach: Routes.getRoutesByRegion($data)">
+                    <tr>
+                        <td data-bind="text: $data.routeName.toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
+                        <td data-bind="text: PokemonFactory.routeHealth($data.number, $parents[0]).toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
+                        <td data-bind="text: Math.floor((Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + 26) + Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + -25))/2).toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
+                        <td data-bind="text: Math.floor(((Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + 26) + Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + -25))/2)*1.5).toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
+                        <td data-bind="text: Math.floor(((Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + 26) + Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + -25))/2)*2.25).toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
```

Comment:
The `data-order` in here seems to be out of place, if anything should be `data-sort` (might both be interchangeable).

If it shouldn't be sorted by the text that's already in the column. it should be referencing the data to use in this column
eg `data-sort: PokemonFactory.routeHealth($data.number, $parents[0])` which is just the number without the locale string transformation.

So for the first column this does nothing as the `routeName` is already the text in the column, should probably be used for the number columns though as they will sometimes not sort correctly in deferent locales.

Reply:
Yeah, that was a test from before I got this branch fully updated. That should be removed by, but still reviewing sorting overall